libsqlite3-sys/sqlite3/wasm-ext.c

```diff
@@ -0,0 +1,207 @@
+// import implementations from android bionic libc
```

Comment:
What's the license on these?

Reply:
@thomcc It appears to be BSD 3-clause, is this acceptable? https://en.wikipedia.org/wiki/Bionic_%28software%29

Reply:
Yes, it would need to include the license somewhere nearby though, but that's not hard to arrange.